web/src/lib/types/user.ts

```diff
@@ -15,6 +15,10 @@ export interface IAccount {
 export class UserResumeSkill {
     skill: string = "";
 
+    public validate() {
+        return !(this.skill.length === 0);
```

Comment:
Not so important, but "not equal" could also be "different" --> !== :)

Just because it's shorter and the `!==` more difficult to miss than the `!( )`.

You have a few more arount the PR. Not important, but if you change one, also change the others.